monkey/tests/unit_tests/monkey_island/cc/setup/test_process_runner.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from monkey_island.cc.setup.mongo_process_runner import MongoDbRunner
+
+
+def test_create_db_dir(monkeypatch, tmpdir):
+    test_dir_name = "test_dir"
+    monkeypatch.setattr("monkey_island.cc.setup.mongo_process_runner.DB_DIR_NAME", test_dir_name)
+    expected_path = os.path.join(tmpdir, test_dir_name)
```

Comment:
This can be moved to a fixture which can be reused by both the test functions:
```python3
@pytest.fixture
def expected_path(monkeypatch, tmpdir):
    monkeypatch.setattr("monkey_island.cc.setup.mongo_process_runner.DB_DIR_NAME", TEST_DIR_NAME)
    expected_path = os.path.join(tmpdir, TEST_DIR_NAME)
    return expected_path
```